wallet/tutorials/javascript-dapp-simple.md

```diff
@@ -0,0 +1,372 @@
+
+- A module bundler, such as [Webpack](https://github.com/webpack/webpack).
+
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+## Steps
+
+### 1. Set up a new project
+
+Your dapp project directory must have the following structure:
+
+```text
+simple-dapp/
+├─ src/
+│  ├─ index.js
+├─ dist/
+│  ├─ index.html
+```
+
+For the dapp to work, your project script `index.js` must:
+
+- [Detect the MetaMask provider.](#2-detect-metamask)
+- [Determine a user's connected network.](#3-determine-a-users-connected-network)
+- [Access an account.](#4-access-an-account)
+
+#### Create a project
+
+Create and navigate into the `simple-dapp` directory using the following commands:
+
+```bash
+mkdir simple-dapp
+cd simple-dapp
+```
+
+:::caution important
+If you import any modules into your project, such as
+[`@metamask/detect-provider`](https://github.com/MetaMask/detect-provider), use a bundler such as
+[Webpack](https://github.com/webpack/webpack) to compile the modules and create an output script
+`dist/main.js`.
+See [Webpack's Getting Started guide](https://webpack.js.org/guides/getting-started/) for more information.
+:::
+
+In the `simple-dapp` directory, create `src` for the source files and `dist` for the distribution files that output from your Webpack builds. 
+Create a JavaScript file in `src`, then create an HTML file in `dist` from the `simple-dapp` root.
+Include the following HTML code example to establish a link to the JavaScript file generated by Webpack:
+
+```jsx title="index.html"
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Simple dapp</title>
+</head>
+<body>
+    <script src="../dist/index.js"></script>  <!-- Links to the JavaScript bundle created by Webpack -->
+</body>
+</html>
+```
+
+#### Install and configure Webpack
+
+Navigate to the root of your `simple-dapp` directory and run the following commands to initialize npm.
+
+```bash 
+npm init -y && npm install --save-dev webpack webpack-cli
+```
+
+Create a `webpack.config.js` in the root of your project with the following configuration:
+
+```jsx title="index.html"
+const path = require('path');
+
+module.exports = {
+    entry: './src/index.js',
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'index.js'
+    },
+    mode: 'development'
+};
+```
+
+Then, edit `package.json` to include a build script:
+
+```json
+"scripts": {
+    "build": "webpack"
+}
+```
+
+When you run `npm run build` from your terminal, Webpack bundles the content in `src/index.js` into `dist/main.js`, which is linked in `index.html`.
+
+### 2. Detect MetaMask
+
+You can use the `@metamask/detect-provider` module to detect the MetaMask Ethereum provider.
+
+#### Install `@metamask/detect-provider`
+
+Use [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) to install
+`@metamask/detect-provider` in your project directory:
```

Comment:
```suggestion
Install the `@metamask/detect-provider` module in your project directory:
```

Maybe also add another reminder here that this method of detecting MetaMask is deprecated and we recommend using EIP-6963 in production.